specification/servicefabricmanagedclusters/ServiceFabricManagedClusters.Management/back-compat.tsp

```diff
@@ -380,3 +380,6 @@ using Microsoft.ServiceFabric;
   armResourceIdentifier[],
   "csharp"
 );
+
+@@scope(OperationStatus.get, "!csharp");
```

Comment:
this should be moved to client.tsp

Reply:
back-compat.tsp will be imported by main.tsp. It is to ensure the swagger equivalence because swagger is emitted from main.tsp. If your previous swagger has `x-ms-client-name`, that corresponding TypeSpec code should be in back-compat.tsp.

client.tsp is only used to generate SDK. It will not be read by swagger emitter, so your generated swagger doesn't have any content from client.tsp. 

It's better to put these into client.tsp since it doean't impact swagger generation at all.